cypress/support/commands.ts

```diff
@@ -0,0 +1,134 @@
+      cy.visit('/');
+      // disable analytics integrations
+      cy.setLocalStorage('chrome:analytics:disable', 'true');
+      cy.setLocalStorage('chrome:segment:disable', 'true');
+
+      cy.wait(1000);
+      // login into the session
+      try {
+        cy.get('#username-verification').type(Cypress.env('E2E_USER'));
+        cy.get('#login-show-step2').click();
+        cy.get('#password').type(Cypress.env('E2E_PASSWORD'));
+        cy.get('#rh-password-verification-submit-button').click();
+      } catch (error) {
+        cy.get('#username').type(Cypress.env('E2E_USER'));
+        cy.get('#password').type(Cypress.env('E2E_PASSWORD'));
+        cy.get('#submit').click();
+      }
+    },
+    { cacheAcrossSpecs: true }
+  );
+});
+
+Cypress.Commands.add('resetToDefaultLayout', () => {
+  cy.intercept(
+    'POST',
+    '**/api/chrome-service/v1/dashboard-templates/*/reset'
+  ).as('resetLayout');
+  cy.get('button')
+    .contains('Reset to default')
+    .click()
+    .get("[data-ouia-component-id='WarningModal-confirm-checkbox']")
+    .click()
+    .get("button[data-ouia-component-id='WarningModal-confirm-button']")
+    .click();
+  cy.wait('@resetLayout').its('response.statusCode').should('eq', 200);
+});
+
+Cypress.Commands.add('dragTotarget', (sourceSelector, targetSelector) => {
+  const source = Cypress.$(sourceSelector);
+  const target = Cypress.$(targetSelector);
+  const { x, y } = target[0].getBoundingClientRect();
+
+  cy.wrap(source)
+    .trigger('mousedown', {
+      which: 1,
+      button: 0,
+      eventConstructor: 'MouseEvent',
+    })
+    .trigger('pointerdown', { which: 1, button: 0 })
+    .trigger('dragstart', { eventConstructor: 'DragEvent', ...source })
+    .trigger('dragover', { eventConstructor: 'DragEvent', ...target })
+    .trigger('mousemove', {
+      clientX: x,
+      clientY: y,
+      eventConstructor: 'MouseEvent',
+      ...target,
+    })
+    .trigger('drop', { eventConstructor: 'DragEvent', ...target })
+    .trigger('mouseup', { which: 1, button: 0, force: true, ...target })
+    .trigger('pointerup', { which: 1, button: 0, ...target });
+});
+
+Cypress.Commands.add('loadLandingPage', () => {
+  cy.login();
+  cy.visit('/');
+
+  cy.intercept(
+    'GET',
+    '**/api/chrome-service/v1/dashboard-templates?dashboard=landingPage'
+  ).as('loadLayout');
+  cy.wait('@loadLayout').its('response.statusCode').should('eq', 200);
+
+  cy.resetToDefaultLayout();
+});
+
+Cypress.Commands.add('removeWidget', (widgetId: string) => {
+  cy.intercept('PATCH', '**/api/chrome-service/v1/dashboard-templates/*').as(
+    'patchLayout'
+  );
+
+  cy.get(`[data-ouia-component-id="${widgetId}"]`).within(() => {
+    cy.get('[aria-label="widget actions menu toggle"]').click();
+  });
+  cy.get('[data-ouia-component-id="remove-widget"]')
+    .click()
+    .wait('@patchLayout');
+  cy.get(`[data-ouia-component-id="${widgetId}]`).should('not.exist');
+});
```

Comment:
These are not needed as they are connected to widgets and landing page.

Reply:
Makes sense, done :)